transport_nantes/transport_nantes/settings_local.py.DEV.template

```diff
@@ -15,6 +15,7 @@ ALLOWED_HOSTS  = ['localhost', '127.0.0.1', '[::1]']
 MORE_INSTALLED_APPS = [
 ]
 
+# NOT AVAILABLE IN PRODUCTION
```

Comment:
I think it's enough to include a reminder that any value defined here but not in `settings.py` is not in general to be considered available to `settings.py`, that this is a basic development config.

Reply:
Something like
```
# Note that variables like BASE_DIR that do not appear
# in settings.py should not be assumed to be available
# in that file.  There exist differences between
# development and staging/production, and this file is
# precisely where those differences are encoded.
```